typescript/infra/src/agents/index.ts

```diff
@@ -139,7 +139,7 @@ export abstract class AgentHelmManager {
             rpcConsensusType: this.rpcConsensusType(chain),
             protocol: metadata.protocol,
             blocks: { reorgPeriod },
-            maxBatchSize: 4,
+            maxBatchSize: 16,
```

Comment:
are we bumping this to 32 everywhere following our conversation earlier?

Reply:
will do